### PR TITLE
#81 - Add capability to be able to fetch the remaining time of the given team

### PR DESCRIPTION
**Pull request by gregito**

### CODE REVIEW

File: lib/data/models/user.dart [hunk 1]
```diff
@@ -18,44 +19,44 @@ class User extends Entity {
...
+        secondClimberName: parsedJson['secondClimberName'] ?? unsetString,
+        userID: parsedJson['id'] ?? parsedJson['userID'] ?? unsetString,
+        teamName: parsedJson['teamName'] ?? unsetString,
+        category: parsedJson['category'] ?? unsetString,
         tenantId: parsedJson['tenantId'],
-        yearId: parsedJson['yearId'] ?? '',
+        yearId: parsedJson['yearId'] ?? unsetString,
         enabled: parsedJson['isEnabled'],
-        isStartDateSet: parsedJson['isStartDateSet'] ?? false);
+        startTime: parsedJson['startTime'] ?? -1);
```
Comment (terjekig) [inline lib/data/models/user.dart:59]: Could this be unsetInt instead of -1 ?

File: lib/data/models/user.dart [hunk 2]
```diff
@@ -106,8 +107,8 @@ class User extends Entity {
     if (userData.containsKey('category')) {
       category = userData['category'];
     }
-    if (userData.containsKey('startDate')) {
-      isStartDateSet = userData['isStartDateSet'] ?? false;
+    if (userData.containsKey('startTime')) {
+      startTime = userData['startTime'] ?? -1;
```
Comment (terjekig) [inline lib/data/models/user.dart:111]: Same, unsetInt instead of -1?

**Review by terjekig — CHANGES_REQUESTED** [review]
insetInt instead of -1 in some places

File: lib/data/models/user.dart (hunk 2 above)
Comment (gregito) [reply lib/data/models/user.dart]: done

File: lib/data/models/user.dart (hunk 1 above)
Comment (gregito) [reply lib/data/models/user.dart]: done

**Review by terjekig — APPROVED** [review]
Looks good, thanks! :)

**Pull request merged**